olad/www/Makefile.mk

```diff
@@ -68,7 +68,56 @@ dist_img_DATA = \
     olad/www/new/img/light_bulb_off.png \
     olad/www/new/img/light_bulb.png \
     olad/www/new/img/logo-mini.png \
-    olad/www/new/img/logo.png
+    olad/www/new/img/logo-mini@2x.png \
+    olad/www/new/img/logo.png \
+    olad/www/new/img/favicons/android-chrome-144x144.png \
```

Comment:
if you want to keep the favicons directory you will have to specify a new variable at the top, something like: 
```make
favicondir = $(www_datadir)/new/img/favicons
```

and the at the begin of the files that you want to end up in the favicons dir you would have to specify:
```make
dist_favicon_DATA = \
```

otherwise autotools will just end up putting them in the ```/new/img``` directory